### PR TITLE
11.0 merge 044c847

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: dockers/posbox/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM itprojectsllc/install-odoo:8.0
```
Comment (yelizariev) [inline dockers/posbox/Dockerfile:1]: Fix version

File: dockers/posbox/README.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
+PosBox Docker
+=============
+
+Use it for development purpose only.
+
+To run this docker you to provide `access to USB devices <https://stackoverflow.com/questions/24225647/docker-any-way-to-give-access-to-host-usb-or-serial-device>`_ of host machine, e.g.::
+
+  # create postgres container
+  docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo --name db-posbox-8.0 postgres:9.5
```
Comment (yelizariev) [inline dockers/posbox/README.rst:9]: fix version

File: dockers/posbox/README.rst [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  docker run \
+  # expose ports:
+  -p 9069:8069 \
+  # remove container after stopping:
+  --rm \
+  # access to usb:
+  --privileged \
+  -v /dev/bus/usb:/dev/bus/usb
+  # link to postgres
+  --link db-posbox-8.0:db \
```
Comment (yelizariev) [inline dockers/posbox/README.rst:20]: fix version

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Comment by iledarn** [comment]
superseded by https://github.com/it-projects-llc/install-odoo/pull/143

**Pull request closed without merging**